src/sentry/auth/providers/saml2.py

```diff
@@ -233,7 +233,7 @@ def dispatch(self, request, organization_slug):
         # SSO response verification failed
         errors = auth.get_errors()
         if errors:
-            message = _('SAML SSO failed: %(reason)') % {'reason': auth.get_last_error_reason()}
+            message = _('SAML SSO failed: {reason}').format(reason=auth.get_last_error_reason())
```

Comment:
`incomplete format` error before. I'm sure it's something trivial about my syntax. Just switched it over to the new-style format without looking much into it.

Reply:
Yeah, because you'd only use `%s` Not `%(reason)`. I only bring this up because I know it goes throuhg a different process and is not just string formatting since this object is a different type and localization does fun things. But if it works, then whatever.